test/backend_test.in.cpp

```diff
@@ -1453,12 +1453,12 @@ NGRAPH_TEST(${BACKEND_NAME}, convert_int32_bool)
 
     // Create some tensors for input/output
     auto a = backend->create_tensor(element::i32, shape);
-    copy_data(a, vector<int32_t>{1, 2, 3, 4});
+    copy_data(a, vector<int32_t>{0, 12, 23, 0});
```

Comment:
We might want to throw in some more extreme values here (e.g., `MAX_INT32 & (~0xFF)`).